wordpress-mobile/WordPress-iOS
# Fix dirty repository after internal build

This is my second attempt to fix an issue where the repository is dirty after a build of the internal scheme :-) 

https://github.com/wordpress-mobile/WordPress-iOS/pull/13008 has a bit more context about the issue.
In this PR, I made a small change to make sure `rake dependencies` doesn't fail if some of the third party tools required to generate the custom icons for the internal build are missing.

### To test:
- Checkout this branch.
- Run `rake dependencies`.
- Open XCode and build the internal scheme.
- Verify that there are no warnings related to missing icons.
- Verify that the repository is clean after the build.

PR submission checklist:
